### PR TITLE
Try to clarify it all

### PR DESCRIPTION
**Pull request by quite**

### CODE REVIEW

File: doc/quickstart.md [hunk 1]
```diff
@@ -5,59 +5,66 @@ USB device.
...
 
-The USB device is then placed correctly in the programming jig, and
-the hatch closed. The USB device can remain in the jig during repeated
-programming and testing cycles. The jig has a cutout to allow for
-touching next to the LED where the touch sensor is located.
+The USB stick can remain in the jig during repeated development,
+programming and testing cycles. The USB stick is then also connected
+to the computer using the provided USB-C cable (use the USB-C-to-A
+adapter if needed). Note that connecting the USB stick to the computer
+is not required for programming it). For this purpose, the jig also
```
Comment (faern) [inline doc/quickstart.md:27]: `)` with no corresponding `(`

File: doc/quickstart.md [hunk 2]
```diff
@@ -5,59 +5,66 @@ USB device.
...
 
-You are now ready to generate the FPGA bitstream including the standard
-firmware, and program the flash on the connected USB device. This should be run
-as your regular non-root user, but the the programming is done (in the
-Makefile) with `sudo tillitis-iceprog` (so sudo is expected be set up).
+You are now ready to generate the FPGA bitstream (including building
+the standard firmware) and program the flash on the USB stick. The
+following should be run as your regular non-root user, but the
+programming is currently done using `sudo` (so sudo is expected to be
+set up for your user; the Makefile runs `sudo tillitis-iceprog …`).
```
Comment (faern) [inline doc/quickstart.md:42]: Would it not be better to recommend the user to set up proper udev rules? "Run this random program you just compiled as root" does not feel optimal IMHO. It's so easy to make the programmer usable as a regular user.

File: doc/quickstart.md [hunk 3]
```diff
@@ -5,59 +5,66 @@ USB device.
...
-and `1207:8887 Tillitis MTA1-USB-V1`.
+Place the Tillitis Key 1 (the USB stick) correctly in the programming
+jig and close the hatch.
 
-The USB device is then placed correctly in the programming jig, and
-the hatch closed. The USB device can remain in the jig during repeated
-programming and testing cycles. The jig has a cutout to allow for
-touching next to the LED where the touch sensor is located.
+The USB stick can remain in the jig during repeated development,
+programming and testing cycles. The USB stick is then also connected
```
[the remark this answers is not in the document]
Comment (quite) [reply doc/quickstart.md:24]: yes!

File: doc/quickstart.md (hunk 2 above)
Comment (quite) [reply doc/quickstart.md]: later..

**Pull request merged**